src/Writers/HTMLWriter.jl

```diff
@@ -275,6 +276,20 @@ function render_search(ctx)
     end
 end
 
+
+# topbar for mobile
+# ------------------------------------------------------------------------------
+
+function render_topbar(ctx, navnode)
+    @tags div ul li span a i
+
+    page_title = string(mdflatten(pagetitle(ctx, navnode)))
+    topbarleft = ul[".float-left"](li(span(page_title)))
+    topbarright = ul[".float-right"](li(a[".btn", :href => "#"](i[".fa .fa-bars", Symbol("aria-hidden") => "true"])))
```

Comment:
just updated, used `span` tag to align the PAGE_TITLE in the css (`<span>{PAGE_TITLE}</span>`).